integration/helpers/vite-plugin-cloudflare-template/wrangler.toml

```diff
@@ -1,5 +1,5 @@
 name = "react-router-app"
-compatibility_date = "2024-11-18"
+compatibility_date = "2025-03-11"
```

Comment:
Could you make sure `@cloudflare/workers-types` dependencies are updated to this version too?

Reply:
Actually, with wrangler 4, you don't even import `@cloudflare/workers-types` anymore. `wrangler types` generates worker-configuration.d.ts and puts all the relevant types you need in it, depending on your configuration (`compatibility_date`, `compatibility_flags`, etc).